app/assets/stylesheets/revised/mixins/_buttons.scss

```diff
@@ -228,7 +228,8 @@ $btn-border-radius-sm: 1px;
 //
 // Base styles
 //
-.container, .container-fluid, .bootstrapstyle {
+
+.container, .container-fluid, .bootstrapstyle, #dirty-dialog {
```

Comment:
the dirty dialog buttons weren't styled. Add it as a wrapper to enable styling them